src/scripts/install.sh

```diff
@@ -1,4 +1,10 @@
-pip3 install aws-sam-cli --user
-echo 'export PATH=$PATH:~/.local/bin' >>$BASH_ENV
-export PATH=$PATH:~/.local/bin
+cd /tmp || true
+if [[ $EUID == 0 ]]; then export SUDO=""; else # Check if we are root
+  export SUDO="sudo";
+fi
+curl -L https://github.com/aws/aws-sam-cli/releases/latest/download/aws-sam-cli-linux-x86_64.zip -o aws-sam-cli-linux-x86_64.zip
```

Comment:
Given our pushes for ARM support, should we include it here in some way?

Reply:
This does not appear to be available: https://github.com/aws/aws-sam-cli/releases
I have added to re existing request here: https://github.com/aws/aws-sam-cli/issues/2713